dm/master/openapi_controller.go

```diff
@@ -511,7 +511,8 @@ func (s *Server) getTask(ctx context.Context, taskName string, req openapi.DMAPI
 	return task, nil
 }
 
-func (s *Server) getTaskStatus(ctx context.Context, taskName string, req openapi.DMAPIGetTaskStatusParams) ([]openapi.SubTaskStatus, error) {
+func (s *Server) getTaskStatus(ctx context.Context, taskName string) ([]openapi.SubTaskStatus, error) {
+	req := openapi.DMAPIGetTaskStatusParams{}
 	if req.SourceNameList == nil || len(*req.SourceNameList) == 0 {
```

Comment:
doesn't this `if` condition becomes unconditionally `true` then?

Reply:
in all cases `req` is `openapi.DMAPIGetTaskStatusParams{}`, so the PR author just push down to here. I guess it's because OpenAPI is not finished developement but we don't have it in plan 😂

Reply:
yes, it's always true i think.